models/sale.coffee

```diff
@@ -116,6 +116,10 @@ module.exports = class Sale extends Backbone.Model
   isRegistrationEnded: ->
     @isAuction() and moment().isAfter(@get 'registration_ends_at')
 
+  isLiveAuction: ->
+    @get('live_start_at') and moment(@get('live_start_at')).isBefore(moment(@get('end_at')))
+
+  # Is there a way to manually end/close out a live sale besides looking at end_at?
```

Comment:
@sweir27 just realized this PR got merged without addressing this point. I'll take a look at it in a few!